examples/text_to_speech_synthesizer/waveglow_handler.py

```diff
@@ -101,20 +102,3 @@ def postprocess(self, inference_output):
             data = output.read()
         os.remove(path)
         return [data]
```

Comment:
Some of the code in `initialize` duplicates what's in BaseHandler. Call `super().initialize(context)` instead of repeating the same code about loading models.